<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 3.2.0">
  <meta name="generator" content="Hugo 0.53" />
  <meta name="author" content="Lorena Pantano">

  
  
  
  
    
  
  <meta name="description" content="This post should show you an easy way to get the following data type integrated into a figure:
 functional enrichment analysis gene expression data from any technology miRNA expression data from any technology  I am using the function isoNetwork from the package isomiRs, that of course is developed by me :) My ego is not that big, it is just I wanted a figure showing that information, and I couldn&rsquo;t find any at a time, but if you know any, tweet me about it to @lopantano.">

  
  <link rel="alternate" hreflang="en-us" href="http://lpantano.github.io/post/legacy/2018-08-13-omic-figure-for-mirna-gene-functional-data/">

  


  

  

  

  

  

  

  
  
  
  <meta name="theme-color" content="#2962ff">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha256-eSi1q2PG6J7g7ib17yAaWMcrr5GrtohYChqibrV7PBE=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" crossorigin="anonymous">
        
      
    

    

    

  

  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:400,400italic,700|Roboto+Mono">
  

  <link rel="stylesheet" href="/styles.css">
  

  
  
  

  
  <link rel="alternate" href="http://lpantano.github.io/index.xml" type="application/rss+xml" title="My bits">
  <link rel="feed" href="http://lpantano.github.io/index.xml" type="application/rss+xml" title="My bits">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="http://lpantano.github.io/post/legacy/2018-08-13-omic-figure-for-mirna-gene-functional-data/">

  
  
  
  
    
  
  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@lopantano">
  <meta property="twitter:creator" content="@lopantano">
  
  <meta property="og:site_name" content="My bits">
  <meta property="og:url" content="http://lpantano.github.io/post/legacy/2018-08-13-omic-figure-for-mirna-gene-functional-data/">
  <meta property="og:title" content="How to plot miRNA, gene expression and functional analysis together | My bits">
  <meta property="og:description" content="This post should show you an easy way to get the following data type integrated into a figure:
 functional enrichment analysis gene expression data from any technology miRNA expression data from any technology  I am using the function isoNetwork from the package isomiRs, that of course is developed by me :) My ego is not that big, it is just I wanted a figure showing that information, and I couldn&rsquo;t find any at a time, but if you know any, tweet me about it to @lopantano."><meta property="og:image" content="http://lpantano.github.io/img/portrait.jpg">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-08-12T09:00:00-04:00">
  
  <meta property="article:modified_time" content="2018-08-12T09:00:00-04:00">
  

  

  

  <title>How to plot miRNA, gene expression and functional analysis together | My bits</title>

</head>
<body id="top" data-spy="scroll" data-target="#TableOfContents" data-offset="71" >
  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" role="textbox" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">My bits</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav ml-auto">
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#publications_selected">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#posts">
            
            <span>Posts</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/talk/">
            
            <span>Talks</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#teaching">
            
            <span>Teaching</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        

      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        

        
        <li class="nav-item">
          <a class="nav-link js-dark-toggle" href="#"><i class="fas fa-moon" aria-hidden="true"></i></a>
        </li>
        

      </ul>

    </div>
  </div>
</nav>


<article class="article" itemscope itemtype="http://schema.org/Article">

  












  

  
  
  
<div class="article-container pt-3">
  <h1 itemprop="name">How to plot miRNA, gene expression and functional analysis together</h1>

  

  
    

<div class="article-metadata">

  
  
  <span itemscope itemprop="author" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Lorena Pantano">
  </span>
  

  <span class="article-date">
    
    <meta content="2018-08-12 09:00:00 -0400 EDT" itemprop="datePublished">
    <time datetime="2018-08-12 09:00:00 -0400 EDT" itemprop="dateModified">
      Aug 12, 2018
    </time>
  </span>
  <span itemscope itemprop="publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Lorena Pantano">
  </span>

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    4 min read
  </span>
  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=How%20to%20plot%20miRNA%2c%20gene%20expression%20and%20functional%20analysis%20together&amp;url=http%3a%2f%2flpantano.github.io%2fpost%2flegacy%2f2018-08-13-omic-figure-for-mirna-gene-functional-data%2f"
         target="_blank" rel="noopener">
        <i class="fab fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=http%3a%2f%2flpantano.github.io%2fpost%2flegacy%2f2018-08-13-omic-figure-for-mirna-gene-functional-data%2f"
         target="_blank" rel="noopener">
        <i class="fab fa-facebook-f"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=http%3a%2f%2flpantano.github.io%2fpost%2flegacy%2f2018-08-13-omic-figure-for-mirna-gene-functional-data%2f&amp;title=How%20to%20plot%20miRNA%2c%20gene%20expression%20and%20functional%20analysis%20together"
         target="_blank" rel="noopener">
        <i class="fab fa-linkedin-in"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=http%3a%2f%2flpantano.github.io%2fpost%2flegacy%2f2018-08-13-omic-figure-for-mirna-gene-functional-data%2f&amp;title=How%20to%20plot%20miRNA%2c%20gene%20expression%20and%20functional%20analysis%20together"
         target="_blank" rel="noopener">
        <i class="fab fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=How%20to%20plot%20miRNA%2c%20gene%20expression%20and%20functional%20analysis%20together&amp;body=http%3a%2f%2flpantano.github.io%2fpost%2flegacy%2f2018-08-13-omic-figure-for-mirna-gene-functional-data%2f">
        <i class="fas fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>

    















  
</div>



  <div class="article-container">

    <div class="article-style" itemprop="articleBody">
      <p>This post should show you an easy way to get the following data type
integrated into a figure:</p>

<ul>
<li>functional enrichment analysis</li>
<li>gene expression data from any technology</li>
<li>miRNA expression data from any technology</li>
</ul>

<p>I am using the function <code>isoNetwork</code> from the package <code>isomiRs</code>, that of
course is developed by me :) My ego is not that big, it is just I wanted
a figure showing that information, and I couldn&rsquo;t find any at a time,
but if you know any, tweet me about it to @lopantano.</p>

<p>This function needs some pre-computed information, like the normalized
expression and the targeted genes by miRNAs. Normally, the significant
genes and miRNAs from a differentially expression analysis would do it.</p>

<p>The first requirement is two have these two datasets into a
SummarizedExperiment object As an example, I am using the published data
from Folic Acid Mouse model (GSE65267) that has been analyzed with
bcbio-nextgen pipeline and bcbioRNASeq package.</p>

<p>I saved these analysis into an R object that contains:</p>

<ul>
<li>fa_mrna: gene expression</li>
<li>fa_mirna: miRNA expression</li>
<li>fa_cold: metadata</li>
</ul>

<p>From that, I ran DESeq2 to get the significant genes and miRNAs from the
comparison day14 vs day0 (normal).</p>

<pre><code>R&gt; load(&quot;fa_isonetwork.rda&quot;)
R&gt; library(isomiRs)
R&gt; library(DESeq2)
R&gt; library(SummarizedExperiment)
R&gt;
R&gt; mi_cold = fa_cold[colnames(fa_mirna), , drop = F]
R&gt; mi_dse = DESeqDataSetFromMatrix(round(2^fa_mirna), mi_cold, design = ~day)
R&gt; mi_dse = DESeq(mi_dse)
R&gt; mi_res = results(mi_dse, name = &quot;day_day14_vs_normal&quot;)
R&gt; mi_res = mi_res[!is.na(mi_res$padj), ]
R&gt; mi_top = row.names(mi_res[mi_res$padj &lt; 0.05, ])
R&gt; mi_rse = SummarizedExperiment(assays = SimpleList(norm = fa_mirna), colData = mi_cold,
+     metadata = list(sign = mi_top))
R&gt;
R&gt; m_cold = fa_cold[colnames(fa_mrna), , drop = F]
R&gt; m_dse = DESeqDataSetFromMatrix(round(2^fa_mrna), m_cold, design = ~day)
R&gt; m_dse = DESeq(m_dse)
R&gt; m_res = results(m_dse, name = &quot;day_day14_vs_normal&quot;)
R&gt; m_res = m_res[!is.na(m_res$padj), ]
R&gt; m_top = row.names(m_res[m_res$padj &lt; 0.05, ])
R&gt; m_rse = SummarizedExperiment(assays = SimpleList(norm = fa_mrna), colData = m_cold,
+     metadata = list(sign = m_top))
</code></pre>

<p>After that, I ran the functional analysis with <a href="http://bioconductor.org/packages/release/bioc/html/clusterProfiler.html" target="_blank"><code>clusterProfiler</code> Bioc
package</a>:</p>

<pre><code>R&gt; library(org.Mm.eg.db)
R&gt; library(clusterProfiler)
R&gt;
R&gt;
R&gt; ego &lt;- enrichGO(m_top, org.Mm.eg.db, &quot;ENSEMBL&quot;, ont = &quot;MF&quot;, universe = rownames(m_res))
</code></pre>

<p>Actually, it could be anything as far as the table have the same naming
that the one coming from this package.</p>

<p>Last step before the real integration analysis, is to get the predicted
targets of the miRNA. This can be done with any package as far as you
get a table with two columns: gene and mirna.</p>

<p>I used two options, one using the already implemented code in the
package (that uses targetscan database):</p>

<pre><code>R&gt; library(targetscan.Mm.eg.db)
R&gt; m2t = mirna2targetscan(mi_top, species = &quot;mmu&quot;, org = org.Mm.eg.db, keytype = &quot;ENSEMBL&quot;)
R&gt;
R&gt; mirna_targets = findTargets(mi_rse, m_rse, m2t[, c(&quot;ENSEMBL&quot;, &quot;mir&quot;)], summarize = &quot;day&quot;,
+     min_cor = -0.7)

## Dimmension of cor matrix: 49 5233
</code></pre>

<p>or using the function <code>get_multimir</code> implemented on <a href="http://bioconductor.org/packages/release/bioc/html/multiMir.html" target="_blank"><code>multiMiR</code> Bioc
package</a>.</p>

<pre><code>R&gt; library(multiMiR)
R&gt; multimir_results &lt;- get_multimir(org = &quot;mmu&quot;, mirna = mi_top, table = &quot;validated&quot;,
+     summary = TRUE)
R&gt;
R&gt; library(magrittr)
R&gt; m2t_multimir = slot(multimir_results, &quot;data&quot;)[, c(&quot;target_ensembl&quot;, &quot;mature_mirna_id&quot;)] %&gt;%
+     dplyr::filter(target_ensembl != &quot;&quot;) %&gt;% dplyr::distinct()
R&gt;
R&gt; mirna_targets = findTargets(mi_rse, m_rse, m2t_multimir, summarize = &quot;day&quot;,
+     min_cor = -0.7)
</code></pre>

<p>In both cases, I ran <code>findTargets</code> to use the expression data to make
sure the correlation between the miRNA and predicted gene is negative
since is the known biological function of miRNAs, being negative
modulators.</p>

<p>Finally, we use <code>isoNetwork</code> to put all the data together, and
<code>isoPlotNet</code> to create the final figure with all information.</p>

<pre><code>R&gt; data &lt;- isoNetwork(mi_rse, m_rse, min_fc = 0.1, summarize = &quot;day&quot;, target = mirna_targets,
+     enrich = ego)

## Dimmension of cor matrix: 49 305

R&gt; isoPlotNet(data, minGenes = 1)
</code></pre>

<p><img src="https://github.com/lpantano/mypubs/raw/master/code-blog/isonetwork/post_files/figure-markdown_strict/unnamed-chunk-5-1.png" /></p>

<p>It is an easy way to spot what pathways contain genes that are targeted
by miRNAs.</p>

<p>Note that this function won&rsquo;t indicate if a pathway is enriched on miRNA
targets. It shows how these three different data set can be put together
to explore the relationship among them.</p>

<hr />

<blockquote>
<p>&ldquo;self-control in bioinformatics: be nice to your collaborator when
they can not open a TSV file with excel.&rdquo;</p>
</blockquote>

    </div>

    


<div class="article-tags">
  
  <a class="badge badge-light" href="/tags/r/">R</a>
  
  <a class="badge badge-light" href="/tags/mirna/">miRNA</a>
  
  <a class="badge badge-light" href="/tags/networks/">networks</a>
  
</div>



    






<div class="media author-card" itemscope itemtype="http://schema.org/Person">
  
  <img class="portrait mr-3" src="/img/portrait.jpg" itemprop="image" alt="Avatar">
  
  <div class="media-body">
    <h5 class="card-title" itemprop="name"><a href="/">Lorena Pantano</a></h5>
    <h6 class="card-subtitle">Research Scientist</h6>
    
    <ul class="network-icon" aria-hidden="true">
      
      
      
      
        
      
      
      
      
      
        
      
      <li>
        <a itemprop="sameAs" href="/#contact" >
          <i class="fas fa-envelope"></i>
        </a>
      </li>
      
      
      
      
        
      
      
      
      
      
        
      
      <li>
        <a itemprop="sameAs" href="https://twitter.com/lopantano" target="_blank" rel="noopener">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
      
      
      
      
      
      
      
      
        
      
      <li>
        <a itemprop="sameAs" href="https://scholar.google.es/citations?user=MeQGlRwAAAAJ&amp;hl=en" target="_blank" rel="noopener">
          <i class="ai ai-google-scholar"></i>
        </a>
      </li>
      
      
      
      
        
      
      
      
      
      
        
      
      <li>
        <a itemprop="sameAs" href="https://github.com/lpantano" target="_blank" rel="noopener">
          <i class="fab fa-github"></i>
        </a>
      </li>
      
    </ul>
  </div>
</div>




    
    
    <div class="article-widget">
      <div class="hr-light"></div>
      <h3>Related</h3>
      <ul>
        
        <li><a href="/talk/bosc2018/">miRTOP: An open source community project for the development of a unified format file for miRNA data</a></li>
        
        <li><a href="/post/legacy/2017-08-24-subset-object-creates-bigger-file/">Subset of object creates bigger RDA file size than original object</a></li>
        
        <li><a href="/post/legacy/2016-03-24-mirna-annotation-tools-comparison/">miRNA Annotation Tools Comparison</a></li>
        
        <li><a href="/post/legacy/2013-10-29-small-rna-abundance-miss-viz/">small RNA abundance [miss-] viz</a></li>
        
        <li><a href="/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/">Get colors for your heatmap annotation</a></li>
        
      </ul>
    </div>
    

    

    


  </div>
</article>

<div class="container">
  <footer class="site-footer">
  

  <p class="powered-by">
    &copy; 2018 &middot; 

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

</div>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha512-+NqPlbbtM1QqiK8ZAo4Yrj2c4lNQoGv8P79DPtKzj++l5jnN39rHA/xsqn8zE9l0uSoxaCdrOgFs6yjyfbBxSg==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha256-VsEqElsCHSGmnmHXGQzvoWjWwoznFSZc6hs7ARLRacQ=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
        
      

      
      
    

    
    

    
    
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'results': "results found",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    
    
    
    
    
    
    
    <script src="/js/academic.min.d037ee5294b166a79dec317c58aea9cc.js"></script>

    

  </body>
</html>

