<!DOCTYPE html>
<html lang="en-us">

<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 4.4.0">

  

  
  
  
  
  
    
    
    
  
  

  <meta name="author" content="Lorena Pantano">

  
  
  
    
  
  <meta name="description" content="This post will show how to configure quickly the colors for the annotation of rows/columns that go on top or on the side of a heatmap.
I normally use pheatmap a lot. Recently I discovered ComplexHeatmap. In both cases I spend always sometime changing the colors of the annotations. I ended up coding a function inside my package DEGreport to do that.
Given a data.frame with metadata data information will do:">

  
  <link rel="alternate" hreflang="en-us" href="http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/">

  


  

  
  
  
  <meta name="theme-color" content="#2962ff">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.0/css/all.css" integrity="sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/github.min.css" crossorigin="anonymous" title="hl-light">
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/styles/dracula.min.css" crossorigin="anonymous" title="hl-dark" disabled>
        
      
    

    

    

  

  
  
  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:400,400italic,700|Roboto+Mono&display=swap">
  

  
  
  
  <link rel="stylesheet" href="/css/academic.min.3bc694af15fd1e4ff7262e7dd46f11a8.css">

  

  
  
  

  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/">

  
  
  
  
    
    
  
  <meta property="twitter:card" content="summary">
  
  <meta property="twitter:site" content="@lopantano">
  <meta property="twitter:creator" content="@lopantano">
  
  <meta property="og:site_name" content="My BioBits">
  <meta property="og:url" content="http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/">
  <meta property="og:title" content="Get colors for your heatmap annotation | My BioBits">
  <meta property="og:description" content="This post will show how to configure quickly the colors for the annotation of rows/columns that go on top or on the side of a heatmap.
I normally use pheatmap a lot. Recently I discovered ComplexHeatmap. In both cases I spend always sometime changing the colors of the annotations. I ended up coding a function inside my package DEGreport to do that.
Given a data.frame with metadata data information will do:"><meta property="og:image" content="http://lpantano.github.io/img/icon-192.png">
  <meta property="twitter:image" content="http://lpantano.github.io/img/icon-192.png"><meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-09-21T10:13:02-04:00">
  
  <meta property="article:modified_time" content="2018-09-21T10:13:02-04:00">
  

  


  





  <title>Get colors for your heatmap annotation | My BioBits</title>

</head>

<body id="top" data-spy="scroll" data-offset="70" data-target="#TableOfContents" >

  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


  
<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0 compensate-for-scrollbar" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">My BioBits</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav mr-auto">
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#about"><span>Home</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#posts"><span>Posts</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#featured"><span>Publications</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#talks"><span>Talks</span></a>
        </li>

        
        

        

        
        
        
          
        

        
        
        
        
        
        
          
          
          
            
          
          
        

        <li class="nav-item">
          <a class="nav-link " href="/#contact"><span>Contact</span></a>
        </li>

        
        

      
      </ul>
      <ul class="navbar-nav ml-auto">
      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        

        
        <li class="nav-item">
          <a class="nav-link js-dark-toggle" href="#"><i class="fas fa-moon" aria-hidden="true"></i></a>
        </li>
        

      </ul>

    </div>
  </div>
</nav>


  <article class="article" itemscope itemtype="http://schema.org/Article">

  












  

  
  
  
<div class="article-container pt-3">
  <h1 itemprop="name">Get colors for your heatmap annotation</h1>

  

  
    



<meta content="2018-09-21 10:13:02 -0400 EDT" itemprop="datePublished">
<meta content="2018-09-21 10:13:02 -0400 EDT" itemprop="dateModified">

<div class="article-metadata">

  
  

  
  <span class="article-date">
    
    
      
    
    <time>Sep 21, 2018</time>
  </span>
  

  

  
  <span class="middot-divider"></span>
  <span class="article-reading-time">
    5 min read
  </span>
  

  
  

  
  

  
    
<div class="share-box" aria-hidden="true">
  <ul class="share">
    
      
      
      
        
      
      
      
      <li>
        <a href="https://twitter.com/intent/tweet?url=http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/&amp;text=Get%20colors%20for%20your%20heatmap%20annotation" target="_blank" rel="noopener" class="share-btn-twitter">
          <i class="fab fa-twitter"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.facebook.com/sharer.php?u=http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/&amp;t=Get%20colors%20for%20your%20heatmap%20annotation" target="_blank" rel="noopener" class="share-btn-facebook">
          <i class="fab fa-facebook-f"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="mailto:?subject=Get%20colors%20for%20your%20heatmap%20annotation&amp;body=http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/" target="_blank" rel="noopener" class="share-btn-email">
          <i class="fas fa-envelope"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://www.linkedin.com/shareArticle?url=http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/&amp;title=Get%20colors%20for%20your%20heatmap%20annotation" target="_blank" rel="noopener" class="share-btn-linkedin">
          <i class="fab fa-linkedin-in"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://web.whatsapp.com/send?text=Get%20colors%20for%20your%20heatmap%20annotation%20http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/" target="_blank" rel="noopener" class="share-btn-whatsapp">
          <i class="fab fa-whatsapp"></i>
        </a>
      </li>
    
      
      
      
        
      
      
      
      <li>
        <a href="https://service.weibo.com/share/share.php?url=http://lpantano.github.io/post/legacy/2018-09-21-colors-for-annotations-for-heatmaps/&amp;title=Get%20colors%20for%20your%20heatmap%20annotation" target="_blank" rel="noopener" class="share-btn-weibo">
          <i class="fab fa-weibo"></i>
        </a>
      </li>
    
  </ul>
</div>


  

</div>

    














  
</div>



  <div class="article-container">

    <div class="article-style" itemprop="articleBody">
      

<p>This post will show how to configure quickly the colors for the annotation of rows/columns that go on top or on the side of a heatmap.</p>

<p>I normally use <code>pheatmap</code> a lot. Recently I discovered <code>ComplexHeatmap</code>. In both cases I spend always sometime changing the colors of the annotations. I ended up coding a function inside my package <code>DEGreport</code> to do that.</p>

<p>Given a data.frame with metadata data information will do:</p>

<ul>
<li>white-black scale for numerical variables</li>
<li>blue-orange for categorical values with only two categories</li>
<li><code>Set2</code> palette for categorical values with more than two categories</li>
</ul>

<p>All three color sets can be changed using the <a href="http://lpantano.github.io/DEGreport/reference/degColors.html" target="_blank">parameters function</a>.</p>

<pre><code class="language-r">library(DEGreport)
library(stemHypoxia)
data(stemHypoxia)
</code></pre>

<p>I am going to use this <a href="https://bioconductor.org/packages/release/data/experiment/html/stemHypoxia.html" target="_blank">stemcell experiment dataset</a> I found randomly in Bioconductor page.</p>

<pre><code class="language-r">exp = as.matrix(M[,3:30])
rownames(exp) = M[,2]
rownames(design) = colnames(exp)
design = design[,c(1:2)]
design$time = as.factor(design$time)
</code></pre>

<p>I&rsquo;ll find the most variable genes.</p>

<pre><code class="language-r">most_variable = names(tail(sort(apply(exp, 1, sd)), 1000))
design$mean_gene = colMeans(exp[most_variable,])
</code></pre>

<p>And we can plot the heatmap with new colors with need to specify each variable at a time.</p>

<pre><code class="language-r">library(pheatmap)
pheatmap(exp[most_variable,], scale = &quot;row&quot;, show_rownames = FALSE,
         annotation_col = design,
         annotation_colors = degColors(design))
</code></pre>

<p><img src="https://github.com/lpantano/mypubs/raw/master/code-blog/degcolor/degcolors_files/figure-markdown_github/unnamed-chunk-4-1.png" alt="" /></p>

<p>If you use <code>ComplexHeatmap</code> you can activate <code>col_fun = TRUE</code> to get it working.</p>

<ul>
<li><p>R Session</p>

<pre><code class="language-r">devtools::session_info()
</code></pre>

<h2 id="warning-description-file-has-an-encoding-field-and-re-encoding-is-not">Warning: &lsquo;DESCRIPTION&rsquo; file has an &lsquo;Encoding&rsquo; field and re-encoding is not</h2>

<h2 id="possible">possible</h2>

<h2 id="session-info">Session info &mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;-</h2>

<h2 id="setting-value">setting  value</h2>

<h2 id="version-r-version-3-5-1-2018-07-02">version  R version 3.5.1 (2018-07-02)</h2>

<h2 id="system-x86-64-darwin15-6-0">system   x86_64, darwin15.6.0</h2>

<h2 id="ui-x11">ui       X11</h2>

<h2 id="language-en">language (EN)</h2>

<h2 id="collate-en-us-utf-8">collate  en_US.UTF-8</h2>

<h2 id="tz-america-new-york">tz       America/New_York</h2>

<h2 id="date-2018-09-21">date     2018-09-21</h2>

<h2 id="packages">Packages &mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&mdash;&ndash;</h2>

<h2 id="package-version-date-source">package              * version   date       source</h2>

<h2 id="acepack-1-4-1-2016-10-29-cran-1-4-1">acepack                1.4.1     2016-10-29 cran (@1.4.1)</h2>

<h2 id="annotate-1-58-0-2018-05-01-cran-1-58-0">annotate               1.58.0    2018-05-01 cran (@1.58.0)</h2>

<h2 id="annotationdbi-1-42-1-2018-05-08-bioconductor">AnnotationDbi          1.42.1    2018-05-08 Bioconductor</h2>

<h2 id="assertthat-0-2-0-2017-04-11-cran-r-3-5-0">assertthat             0.2.0     2017-04-11 CRAN (R 3.5.0)</h2>

<h2 id="backports-1-1-2-2017-12-13-cran-r-3-5-0">backports              1.1.2     2017-12-13 CRAN (R 3.5.0)</h2>

<h2 id="base-3-5-1-2018-07-05-local">base                 * 3.5.1     2018-07-05 local</h2>

<h2 id="base64enc-0-1-3-2015-07-28-cran-r-3-5-0">base64enc              0.1-3     2015-07-28 CRAN (R 3.5.0)</h2>

<h2 id="bindr-0-1-1-2018-03-13-cran-r-3-5-0">bindr                  0.1.1     2018-03-13 CRAN (R 3.5.0)</h2>

<h2 id="bindrcpp-0-2-2-2018-03-29-cran-r-3-5-0">bindrcpp               0.2.2     2018-03-29 CRAN (R 3.5.0)</h2>

<h2 id="biobase-2-40-0-2018-05-01-bioconductor">Biobase                2.40.0    2018-05-01 Bioconductor</h2>

<h2 id="biocgenerics-0-26-0-2018-05-01-bioconductor">BiocGenerics           0.26.0    2018-05-01 Bioconductor</h2>

<h2 id="biocparallel-1-14-2-2018-07-08-cran-1-14-2">BiocParallel           1.14.2    2018-07-08 cran (@1.14.2)</h2>

<h2 id="bit-1-1-14-2018-05-29-cran-r-3-5-0">bit                    1.1-14    2018-05-29 CRAN (R 3.5.0)</h2>

<h2 id="bit64-0-9-7-2017-05-08-cran-r-3-5-0">bit64                  0.9-7     2017-05-08 CRAN (R 3.5.0)</h2>

<h2 id="bitops-1-0-6-2013-08-17-cran-1-0-6">bitops                 1.0-6     2013-08-17 cran (@1.0-6)</h2>

<h2 id="blob-1-1-1-2018-03-25-cran-r-3-5-0">blob                   1.1.1     2018-03-25 CRAN (R 3.5.0)</h2>

<h2 id="broom-0-5-0-2018-07-17-cran-r-3-5-0">broom                  0.5.0     2018-07-17 CRAN (R 3.5.0)</h2>

<h2 id="checkmate-1-8-5-2017-10-24-cran-1-8-5">checkmate              1.8.5     2017-10-24 cran (@1.8.5)</h2>

<h2 id="circlize-0-4-4-2018-06-10-cran-0-4-4">circlize               0.4.4     2018-06-10 cran (@0.4.4)</h2>

<h2 id="cluster-2-0-7-1-2018-04-13-cran-r-3-5-1">cluster                2.0.7-1   2018-04-13 CRAN (R 3.5.1)</h2>

<h2 id="colorspace-1-3-2-2016-12-14-cran-r-3-5-0">colorspace             1.3-2     2016-12-14 CRAN (R 3.5.0)</h2>

<h2 id="compiler-3-5-1-2018-07-05-local">compiler               3.5.1     2018-07-05 local</h2>

<h2 id="complexheatmap-1-18-1-2018-06-19-cran-1-18-1">ComplexHeatmap         1.18.1    2018-06-19 cran (@1.18.1)</h2>

<h2 id="consensusclusterplus-1-44-0-2018-05-01-cran-1-44-0">ConsensusClusterPlus   1.44.0    2018-05-01 cran (@1.44.0)</h2>

<h2 id="cowplot-0-9-3-2018-07-15-cran-0-9-3">cowplot                0.9.3     2018-07-15 cran (@0.9.3)</h2>

<h2 id="crayon-1-3-4-2017-09-16-cran-r-3-5-0">crayon                 1.3.4     2017-09-16 CRAN (R 3.5.0)</h2>

<h2 id="data-table-1-11-4-2018-05-27-cran-1-11-4">data.table             1.11.4    2018-05-27 cran (@1.11.4)</h2>

<h2 id="datasets-3-5-1-2018-07-05-local">datasets             * 3.5.1     2018-07-05 local</h2>

<h2 id="dbi-1-0-0-2018-05-02-cran-r-3-5-0">DBI                    1.0.0     2018-05-02 CRAN (R 3.5.0)</h2>

<h2 id="degreport-1-17-5-2018-09-04-local-lpantano-degreport-na">DEGreport            * 1.17.5    2018-09-04 local (lpantano/DEGreport@NA)</h2>

<h2 id="delayedarray-0-6-5-2018-08-15-cran-0-6-5">DelayedArray           0.6.5     2018-08-15 cran (@0.6.5)</h2>

<h2 id="deseq2-1-20-0-2018-05-01-bioconductor">DESeq2                 1.20.0    2018-05-01 Bioconductor</h2>

<h2 id="devtools-1-13-6-2018-06-27-cran-r-3-5-0">devtools               1.13.6    2018-06-27 CRAN (R 3.5.0)</h2>

<h2 id="digest-0-6-16-2018-08-22-cran-r-3-5-0">digest                 0.6.16    2018-08-22 CRAN (R 3.5.0)</h2>

<h2 id="dplyr-0-7-6-2018-06-29-cran-r-3-5-1">dplyr                  0.7.6     2018-06-29 CRAN (R 3.5.1)</h2>

<h2 id="edger-3-22-3-2018-06-21-cran-3-22-3">edgeR                  3.22.3    2018-06-21 cran (@3.22.3)</h2>

<h2 id="evaluate-0-11-2018-07-17-cran-r-3-5-0">evaluate               0.11      2018-07-17 CRAN (R 3.5.0)</h2>

<h2 id="foreign-0-8-71-2018-07-20-cran-r-3-5-0">foreign                0.8-71    2018-07-20 CRAN (R 3.5.0)</h2>

<h2 id="formula-1-2-3-2018-05-03-cran-1-2-3">Formula                1.2-3     2018-05-03 cran (@1.2-3)</h2>

<h2 id="genefilter-1-62-0-2018-05-01-cran-1-62-0">genefilter             1.62.0    2018-05-01 cran (@1.62.0)</h2>

<h2 id="geneplotter-1-58-0-2018-05-01-cran-1-58-0">geneplotter            1.58.0    2018-05-01 cran (@1.58.0)</h2>

<h2 id="genomeinfodb-1-16-0-2018-05-01-cran-1-16-0">GenomeInfoDb           1.16.0    2018-05-01 cran (@1.16.0)</h2>

<h2 id="genomeinfodbdata-1-1-0-2018-08-31-bioconductor">GenomeInfoDbData       1.1.0     2018-08-31 Bioconductor</h2>

<h2 id="genomicranges-1-32-6-2018-07-20-cran-1-32-6">GenomicRanges          1.32.6    2018-07-20 cran (@1.32.6)</h2>

<h2 id="getoptlong-0-1-7-2018-06-10-cran-0-1-7">GetoptLong             0.1.7     2018-06-10 cran (@0.1.7)</h2>

<h2 id="ggdendro-0-1-20-2016-04-27-cran-0-1-20">ggdendro               0.1-20    2016-04-27 cran (@0.1-20)</h2>

<h2 id="ggplot2-3-0-0-2018-07-03-cran-r-3-5-0">ggplot2                3.0.0     2018-07-03 CRAN (R 3.5.0)</h2>

<h2 id="ggrepel-0-8-0-2018-05-09-cran-0-8-0">ggrepel                0.8.0     2018-05-09 cran (@0.8.0)</h2>

<h2 id="globaloptions-0-1-0-2018-06-09-cran-0-1-0">GlobalOptions          0.1.0     2018-06-09 cran (@0.1.0)</h2>

<h2 id="glue-1-3-0-2018-07-17-cran-r-3-5-0">glue                   1.3.0     2018-07-17 CRAN (R 3.5.0)</h2>

<h2 id="graphics-3-5-1-2018-07-05-local">graphics             * 3.5.1     2018-07-05 local</h2>

<h2 id="grdevices-3-5-1-2018-07-05-local">grDevices            * 3.5.1     2018-07-05 local</h2>

<h2 id="grid-3-5-1-2018-07-05-local">grid                   3.5.1     2018-07-05 local</h2>

<h2 id="gridextra-2-3-2017-09-09-cran-2-3">gridExtra              2.3       2017-09-09 cran (@2.3)</h2>

<h2 id="gtable-0-2-0-2016-02-26-cran-r-3-5-0">gtable                 0.2.0     2016-02-26 CRAN (R 3.5.0)</h2>

<h2 id="hmisc-4-1-1-2018-01-03-cran-4-1-1">Hmisc                  4.1-1     2018-01-03 cran (@4.1-1)</h2>

<h2 id="htmltable-1-12-2018-05-26-cran-1-12">htmlTable              1.12      2018-05-26 cran (@1.12)</h2>

<h2 id="htmltools-0-3-6-2017-04-28-cran-r-3-5-0">htmltools              0.3.6     2017-04-28 CRAN (R 3.5.0)</h2>

<h2 id="htmlwidgets-1-2-2018-04-19-cran-1-2">htmlwidgets            1.2       2018-04-19 cran (@1.2)</h2>

<h2 id="iranges-2-14-11-2018-08-24-bioconductor">IRanges                2.14.11   2018-08-24 Bioconductor</h2>

<h2 id="knitr-1-20-2018-02-20-cran-r-3-5-0">knitr                  1.20      2018-02-20 CRAN (R 3.5.0)</h2>

<h2 id="lasso2-1-2-19-2014-05-31-cran-1-2-19">lasso2                 1.2-19    2014-05-31 cran (@1.2-19)</h2>

<h2 id="lattice-0-20-35-2017-03-25-cran-r-3-5-1">lattice                0.20-35   2017-03-25 CRAN (R 3.5.1)</h2>

<h2 id="latticeextra-0-6-28-2016-02-09-cran-0-6-28">latticeExtra           0.6-28    2016-02-09 cran (@0.6-28)</h2>

<h2 id="lazyeval-0-2-1-2017-10-29-cran-r-3-5-0">lazyeval               0.2.1     2017-10-29 CRAN (R 3.5.0)</h2>

<h2 id="limma-3-36-3-2018-08-25-cran-3-36-3">limma                  3.36.3    2018-08-25 cran (@3.36.3)</h2>

<h2 id="locfit-1-5-9-1-2013-04-20-cran-1-5-9-1">locfit                 1.5-9.1   2013-04-20 cran (@1.5-9.1)</h2>

<h2 id="logging-0-7-103-2013-04-12-cran-0-7-103">logging                0.7-103   2013-04-12 cran (@0.7-103)</h2>

<h2 id="magrittr-1-5-2014-11-22-cran-r-3-5-0">magrittr               1.5       2014-11-22 CRAN (R 3.5.0)</h2>

<h2 id="mass-7-3-50-2018-04-30-cran-r-3-5-1">MASS                   7.3-50    2018-04-30 CRAN (R 3.5.1)</h2>

<h2 id="matrix-1-2-14-2018-04-13-cran-r-3-5-1">Matrix                 1.2-14    2018-04-13 CRAN (R 3.5.1)</h2>

<h2 id="matrixstats-0-54-0-2018-07-23-cran-0-54-0">matrixStats            0.54.0    2018-07-23 cran (@0.54.0)</h2>

<h2 id="memoise-1-1-0-2017-04-21-cran-r-3-5-0">memoise                1.1.0     2017-04-21 CRAN (R 3.5.0)</h2>

<h2 id="methods-3-5-1-2018-07-05-local">methods              * 3.5.1     2018-07-05 local</h2>

<h2 id="mnormt-1-5-5-2016-10-15-cran-1-5-5">mnormt                 1.5-5     2016-10-15 cran (@1.5-5)</h2>

<h2 id="munsell-0-5-0-2018-06-12-cran-r-3-5-0">munsell                0.5.0     2018-06-12 CRAN (R 3.5.0)</h2>

<h2 id="nlme-3-1-137-2018-04-07-cran-r-3-5-1">nlme                   3.1-137   2018-04-07 CRAN (R 3.5.1)</h2>

<h2 id="nnet-7-3-12-2016-02-02-cran-r-3-5-1">nnet                   7.3-12    2016-02-02 CRAN (R 3.5.1)</h2>

<h2 id="nozzle-r1-1-1-1-2013-05-15-cran-1-1-1">Nozzle.R1              1.1-1     2013-05-15 cran (@1.1-1)</h2>

<h2 id="parallel-3-5-1-2018-07-05-local">parallel               3.5.1     2018-07-05 local</h2>

<h2 id="pheatmap-1-0-10-2018-05-19-cran-r-3-5-0">pheatmap             * 1.0.10    2018-05-19 CRAN (R 3.5.0)</h2>

<h2 id="pillar-1-3-0-2018-07-14-cran-r-3-5-0">pillar                 1.3.0     2018-07-14 CRAN (R 3.5.0)</h2>

<h2 id="pkgconfig-2-0-2-2018-08-16-cran-r-3-5-0">pkgconfig              2.0.2     2018-08-16 CRAN (R 3.5.0)</h2>

<h2 id="plyr-1-8-4-2016-06-08-cran-r-3-5-0">plyr                   1.8.4     2016-06-08 CRAN (R 3.5.0)</h2>

<h2 id="psych-1-8-4-2018-05-06-cran-1-8-4">psych                  1.8.4     2018-05-06 cran (@1.8.4)</h2>

<h2 id="purrr-0-2-5-2018-05-29-cran-r-3-5-0">purrr                  0.2.5     2018-05-29 CRAN (R 3.5.0)</h2>

<h2 id="r6-2-2-2-2017-06-17-cran-r-3-5-0">R6                     2.2.2     2017-06-17 CRAN (R 3.5.0)</h2>

<h2 id="rcolorbrewer-1-1-2-2014-12-07-cran-r-3-5-0">RColorBrewer           1.1-2     2014-12-07 CRAN (R 3.5.0)</h2>

<h2 id="rcpp-0-12-18-2018-07-23-cran-r-3-5-0">Rcpp                   0.12.18   2018-07-23 CRAN (R 3.5.0)</h2>

<h2 id="rcurl-1-95-4-11-2018-07-15-cran-1-95-4">RCurl                  1.95-4.11 2018-07-15 cran (@1.95-4.)</h2>

<h2 id="reshape-0-8-7-2017-08-06-cran-0-8-7">reshape                0.8.7     2017-08-06 cran (@0.8.7)</h2>

<h2 id="rjson-0-2-20-2018-06-08-cran-0-2-20">rjson                  0.2.20    2018-06-08 cran (@0.2.20)</h2>

<h2 id="rlang-0-2-2-2018-08-16-cran-r-3-5-0">rlang                  0.2.2     2018-08-16 CRAN (R 3.5.0)</h2>

<h2 id="rmarkdown-1-10-2018-06-11-cran-r-3-5-0">rmarkdown              1.10      2018-06-11 CRAN (R 3.5.0)</h2>

<h2 id="rpart-4-1-13-2018-02-23-cran-r-3-5-1">rpart                  4.1-13    2018-02-23 CRAN (R 3.5.1)</h2>

<h2 id="rprojroot-1-3-2-2018-01-03-cran-r-3-5-0">rprojroot              1.3-2     2018-01-03 CRAN (R 3.5.0)</h2>

<h2 id="rsqlite-2-1-1-2018-05-06-cran-r-3-5-0">RSQLite                2.1.1     2018-05-06 CRAN (R 3.5.0)</h2>

<h2 id="rstudioapi-0-7-2017-09-07-cran-r-3-5-0">rstudioapi             0.7       2017-09-07 CRAN (R 3.5.0)</h2>

<h2 id="s4vectors-0-18-3-2018-06-08-bioconductor">S4Vectors              0.18.3    2018-06-08 Bioconductor</h2>

<h2 id="scales-1-0-0-2018-08-09-cran-r-3-5-0">scales                 1.0.0     2018-08-09 CRAN (R 3.5.0)</h2>

<h2 id="shape-1-4-4-2018-02-07-cran-1-4-4">shape                  1.4.4     2018-02-07 cran (@1.4.4)</h2>

<h2 id="splines-3-5-1-2018-07-05-local">splines                3.5.1     2018-07-05 local</h2>

<h2 id="stats-3-5-1-2018-07-05-local">stats                * 3.5.1     2018-07-05 local</h2>

<h2 id="stats4-3-5-1-2018-07-05-local">stats4                 3.5.1     2018-07-05 local</h2>

<h2 id="stemhypoxia-1-16-0-2018-09-21-bioconductor">stemHypoxia          * 1.16.0    2018-09-21 Bioconductor</h2>

<h2 id="stringi-1-2-4-2018-07-20-cran-r-3-5-0">stringi                1.2.4     2018-07-20 CRAN (R 3.5.0)</h2>

<h2 id="stringr-1-3-1-2018-05-10-cran-r-3-5-0">stringr                1.3.1     2018-05-10 CRAN (R 3.5.0)</h2>

<h2 id="summarizedexperiment-1-10-1-2018-05-11-bioconductor">SummarizedExperiment   1.10.1    2018-05-11 Bioconductor</h2>

<h2 id="survival-2-42-6-2018-07-13-cran-r-3-5-0">survival               2.42-6    2018-07-13 CRAN (R 3.5.0)</h2>

<h2 id="tibble-1-4-2-2018-01-22-cran-r-3-5-0">tibble                 1.4.2     2018-01-22 CRAN (R 3.5.0)</h2>

<h2 id="tidyr-0-8-1-2018-05-18-cran-r-3-5-0">tidyr                  0.8.1     2018-05-18 CRAN (R 3.5.0)</h2>

<h2 id="tidyselect-0-2-4-2018-02-26-cran-r-3-5-0">tidyselect             0.2.4     2018-02-26 CRAN (R 3.5.0)</h2>

<h2 id="tools-3-5-1-2018-07-05-local">tools                  3.5.1     2018-07-05 local</h2>

<h2 id="utils-3-5-1-2018-07-05-local">utils                * 3.5.1     2018-07-05 local</h2>

<h2 id="withr-2-1-2-2018-03-15-cran-r-3-5-0">withr                  2.1.2     2018-03-15 CRAN (R 3.5.0)</h2>

<h2 id="xml-3-98-1-16-2018-08-19-cran-3-98-1">XML                    3.98-1.16 2018-08-19 cran (@3.98-1.)</h2>

<h2 id="xtable-1-8-3-2018-08-29-cran-1-8-3">xtable                 1.8-3     2018-08-29 cran (@1.8-3)</h2>

<h2 id="xvector-0-20-0-2018-05-01-cran-0-20-0">XVector                0.20.0    2018-05-01 cran (@0.20.0)</h2>

<h2 id="yaml-2-2-0-2018-07-25-cran-r-3-5-0">yaml                   2.2.0     2018-07-25 CRAN (R 3.5.0)</h2>

<h2 id="zlibbioc-1-26-0-2018-05-01-cran-1-26-0">zlibbioc               1.26.0    2018-05-01 cran (@1.26.0)</h2></li>
</ul>

    </div>

    

<div class="article-tags">
  
  <a class="badge badge-light" href="/tags/r/">R</a>
  
  <a class="badge badge-light" href="/tags/visualization/">Visualization</a>
  
</div>



    
      








  





  
  
  
    
  
  
  <div class="media author-card" itemscope itemtype="http://schema.org/Person">
    
      
      <img class="portrait mr-3" src="/authors/admin/avatar_hua735c4bd12b5368408d6d400f29b21ec_801139_250x250_fill_q90_lanczos_center.jpg" itemprop="image" alt="Avatar">
    

    <div class="media-body">
      <h5 class="card-title" itemprop="name"><a href="http://lpantano.github.io/">Lorena Pantano</a></h5>
      <h6 class="card-subtitle">Senior Computational Biologist</h6>
      <p class="card-text" itemprop="description">My research interests include distributed robotics, mobile computing and programmable matter.</p>
      <ul class="network-icon" aria-hidden="true">
        
          
          
          
            
          
          
          
          
          
            
          
          <li>
            <a itemprop="sameAs" href="/#contact" >
              <i class="fas fa-envelope"></i>
            </a>
          </li>
        
          
          
          
            
          
          
          
          
          
            
          
          <li>
            <a itemprop="sameAs" href="https://twitter.com/lopantano" target="_blank" rel="noopener">
              <i class="fab fa-twitter"></i>
            </a>
          </li>
        
          
          
          
          
          
          
          
            
          
          <li>
            <a itemprop="sameAs" href="https://scholar.google.co.uk/citations?user=MeQGlRwAAAAJ&amp;hl" target="_blank" rel="noopener">
              <i class="ai ai-google-scholar"></i>
            </a>
          </li>
        
          
          
          
            
          
          
          
          
          
            
          
          <li>
            <a itemprop="sameAs" href="https://github.com/lpantano" target="_blank" rel="noopener">
              <i class="fab fa-github"></i>
            </a>
          </li>
        
      </ul>
    </div>
  </div>



      
      
      <div class="article-widget">
        <div class="hr-light"></div>
        <h3>Related</h3>
        <ul>
          
          <li><a href="/post/legacy/2018-08-13-omic-figure-for-mirna-gene-functional-data/">How to plot miRNA, gene expression and functional analysis together</a></li>
          
          <li><a href="/post/legacy/2017-08-24-subset-object-creates-bigger-file/">Subset of object creates bigger RDA file size than original object</a></li>
          
        </ul>
      </div>
      
    

    

    


  </div>
</article>

      

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.4/imagesloaded.pkgd.min.js" integrity="sha256-lqvxZrPLtfffUl2G/e7szqSvPBILGbwmsGE1MKlOi0Q=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.6/isotope.pkgd.min.js" integrity="sha256-CBrpuqrMhXwcLLUd5tvQ4euBHCdh7wGlDfNz8vbu/iI=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.15.6/highlight.min.js" integrity="sha256-aYTdUrn6Ow1DDgh5JTc3aDGnnju48y/1c8s1dgkYPQ8=" crossorigin="anonymous"></script>
        
      

      
      
    

    
    

    
    
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'results': "results found",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    
    
    
    
    
    
    
    
      
    
    
    
    
    <script src="/js/academic.min.16bbb3750feb7244c9bc409a5a4fe678.js"></script>

    






  
  <div class="container">
    <footer class="site-footer">
  

  <p class="powered-by">
    

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

  </div>
  

  
<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

</body>
</html>
