<!DOCTYPE html>
<html lang="en-us">
<head>

  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="generator" content="Source Themes Academic 3.2.0">
  <meta name="generator" content="Hugo 0.55.3" />
  <meta name="author" content="Lorena Pantano">

  
  
  
  
    
  
  <meta name="description" content="MicroRNAs (miRNAs) are small RNA molecules (20-27 nt long) that are involved in eukaryotic  gene regulation. They regulate targeted genes by RNA complementarity, generally between the miRNA seed region and the 3’ UTR of messenger RNAs. They have become relevant to the study of developmental stages and diseases. With the advance of sequencing technology, the cost of detecting the miRNA transcriptome has decreased substantially, and has led to the discovery of isomiRs (with slight sequence variations relative to the annotated miRNAs). As a consequence, the amount of miRNA data has increased exponentially in the last 5 years, together with the number of pipelines to analyze it. However, there is still a lack of consensus standards in storing and sharing such data, generating diverse output files; hindering comparison between tools, data sharing, and development of downstream analyses independent of the tools used for miRNA detection and quantification. Here, we present a community based, open source project to work toward the standardization of miRNA pipelines and encourage the development of downstream tools for visualization, differential expression, sample clustering, and model prediction analyses. This project is an international collaboration, with experts from different countries that have been developing miRNA pipelines and resources. We have described a standard format for the output of miRNA detection and quantification tools using small RNA-seq data. The format is based on the GFF3 standard in order to support reference coordinates and parent/child relationships between the features. The format contains information foreach sequence and its annotation to the miRNA precursor, the definition of reference or isomiR sequence, its quality, the isomiR type, and abundance in the data set. Moreover, we support a command line python tool to manage the miRNA GFF3 format (miRTop). Currently, miRTop can convert the output of commonly used small RNA-Seq pipelines, such as seqbuster, isomiR-SEA, sRNAbench, and Prost!, as well as BAM files to the miRNA GFF3 format. Importantly, the miRge pipeline has adapted the GFF3 format natively. miRTop can convert miRNA GFF3 files to a count matrix that can be easily imported to any downstream tool (i.e. for differential expression analysis). Furthermore, the tool can export the miRNA GFF file to the isomiRs package, improving the usability of this tool and rendering it independent of the upstream methods used for quantifying miRNA sequences. Together, we believe that the miRTop project will improve accessibility and uniformity of the results oft miRNA pipelines, the possibility to easily run benchmarking analyses, and promote the development of miRNA tools downstream of the detection and quantification steps.">

  
  <link rel="alternate" hreflang="en-us" href="http://lpantano.github.io/talk/bosc2018/">

  


  

  

  

  

  

  

  
  
  
  <meta name="theme-color" content="#2962ff">
  

  
  
  
  
    
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha256-eSi1q2PG6J7g7ib17yAaWMcrr5GrtohYChqibrV7PBE=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/academicons/1.8.6/css/academicons.min.css" integrity="sha256-uFVgMKfistnJAfoCUQigIl+JfUaP47GrRKjf6CTPVmw=" crossorigin="anonymous">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.4.1/css/all.css" integrity="sha384-5sAR7xN1Nv6T6+dT2mhtzEpVJvfS3NScPQTrOxhwjIuvcA67KV2R5Jz6kr4abQsz" crossorigin="anonymous">
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.css" integrity="sha256-ygkqlh3CYSUri3LhQxzdcm0n1EQvH2Y+U5S2idbLtxs=" crossorigin="anonymous">

    
    
    
      
    
    
      
      
        
          <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css" crossorigin="anonymous">
        
      
    

    

    

  

  
  
  <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Montserrat:400,700|Roboto:400,400italic,700|Roboto+Mono">
  

  <link rel="stylesheet" href="/styles.css">
  

  
  
  

  
  <link rel="alternate" href="http://lpantano.github.io/index.xml" type="application/rss+xml" title="My bits">
  <link rel="feed" href="http://lpantano.github.io/index.xml" type="application/rss+xml" title="My bits">
  

  <link rel="manifest" href="/site.webmanifest">
  <link rel="icon" type="image/png" href="/img/icon.png">
  <link rel="apple-touch-icon" type="image/png" href="/img/icon-192.png">

  <link rel="canonical" href="http://lpantano.github.io/talk/bosc2018/">

  
  
  
  
    
  
  <meta property="twitter:card" content="summary_large_image">
  
  <meta property="twitter:site" content="@lopantano">
  <meta property="twitter:creator" content="@lopantano">
  
  <meta property="og:site_name" content="My bits">
  <meta property="og:url" content="http://lpantano.github.io/talk/bosc2018/">
  <meta property="og:title" content="miRTOP: An open source community project for the development of a unified format file for miRNA data | My bits">
  <meta property="og:description" content="MicroRNAs (miRNAs) are small RNA molecules (20-27 nt long) that are involved in eukaryotic  gene regulation. They regulate targeted genes by RNA complementarity, generally between the miRNA seed region and the 3’ UTR of messenger RNAs. They have become relevant to the study of developmental stages and diseases. With the advance of sequencing technology, the cost of detecting the miRNA transcriptome has decreased substantially, and has led to the discovery of isomiRs (with slight sequence variations relative to the annotated miRNAs). As a consequence, the amount of miRNA data has increased exponentially in the last 5 years, together with the number of pipelines to analyze it. However, there is still a lack of consensus standards in storing and sharing such data, generating diverse output files; hindering comparison between tools, data sharing, and development of downstream analyses independent of the tools used for miRNA detection and quantification. Here, we present a community based, open source project to work toward the standardization of miRNA pipelines and encourage the development of downstream tools for visualization, differential expression, sample clustering, and model prediction analyses. This project is an international collaboration, with experts from different countries that have been developing miRNA pipelines and resources. We have described a standard format for the output of miRNA detection and quantification tools using small RNA-seq data. The format is based on the GFF3 standard in order to support reference coordinates and parent/child relationships between the features. The format contains information foreach sequence and its annotation to the miRNA precursor, the definition of reference or isomiR sequence, its quality, the isomiR type, and abundance in the data set. Moreover, we support a command line python tool to manage the miRNA GFF3 format (miRTop). Currently, miRTop can convert the output of commonly used small RNA-Seq pipelines, such as seqbuster, isomiR-SEA, sRNAbench, and Prost!, as well as BAM files to the miRNA GFF3 format. Importantly, the miRge pipeline has adapted the GFF3 format natively. miRTop can convert miRNA GFF3 files to a count matrix that can be easily imported to any downstream tool (i.e. for differential expression analysis). Furthermore, the tool can export the miRNA GFF file to the isomiRs package, improving the usability of this tool and rendering it independent of the upstream methods used for quantifying miRNA sequences. Together, we believe that the miRTop project will improve accessibility and uniformity of the results oft miRNA pipelines, the possibility to easily run benchmarking analyses, and promote the development of miRNA tools downstream of the detection and quantification steps."><meta property="og:image" content="http://lpantano.github.io/img/portrait.jpg">
  <meta property="og:locale" content="en-us">
  
  <meta property="article:published_time" content="2018-06-27T13:13:02-09:00">
  
  <meta property="article:modified_time" content="2018-06-27T13:13:02-09:00">
  

  

  

  <title>miRTOP: An open source community project for the development of a unified format file for miRNA data | My bits</title>

</head>
<body id="top" data-spy="scroll" data-target="#TableOfContents" data-offset="71" >
  <aside class="search-results" id="search">
  <div class="container">
    <section class="search-header">

      <div class="row no-gutters justify-content-between mb-3">
        <div class="col-6">
          <h1>Search</h1>
        </div>
        <div class="col-6 col-search-close">
          <a class="js-search" href="#"><i class="fas fa-times-circle text-muted" aria-hidden="true"></i></a>
        </div>
      </div>

      <div id="search-box">
        
        <input name="q" id="search-query" placeholder="Search..." autocapitalize="off"
        autocomplete="off" autocorrect="off" role="textbox" spellcheck="false" type="search">
        
      </div>

    </section>
    <section class="section-search-results">

      <div id="search-hits">
        
      </div>

    </section>
  </div>
</aside>


<nav class="navbar navbar-light fixed-top navbar-expand-lg py-0" id="navbar-main">
  <div class="container">

    
      <a class="navbar-brand" href="/">My bits</a>
      
      <button type="button" class="navbar-toggler" data-toggle="collapse"
              data-target="#navbar" aria-controls="navbar" aria-expanded="false" aria-label="Toggle navigation">
        <span><i class="fas fa-bars"></i></span>
      </button>
      

    
    <div class="collapse navbar-collapse" id="navbar">

      
      
      <ul class="navbar-nav ml-auto">
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#about">
            
            <span>Home</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#publications_selected">
            
            <span>Publications</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#posts">
            
            <span>Posts</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/talk/">
            
            <span>Talks</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#teaching">
            
            <span>Teaching</span>
            
          </a>
        </li>

        
        

        

        
        
        
          
        

        <li class="nav-item">
          <a class="nav-link" href="/#contact">
            
            <span>Contact</span>
            
          </a>
        </li>

        
        

      

        

        
        <li class="nav-item">
          <a class="nav-link js-search" href="#"><i class="fas fa-search" aria-hidden="true"></i></a>
        </li>
        

        

        
        <li class="nav-item">
          <a class="nav-link js-dark-toggle" href="#"><i class="fas fa-moon" aria-hidden="true"></i></a>
        </li>
        

      </ul>

    </div>
  </div>
</nav>

<div class="pub" itemscope itemtype="http://schema.org/Event">

  












  

  
  
  
<div class="article-container pt-3">
  <h1 itemprop="name">miRTOP: An open source community project for the development of a unified format file for miRNA data</h1>

  

  
    

<div class="article-metadata">

  
  
  
  <div>
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Thomas Desvignes</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Karen EIlbeck</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Ioannis S. Vlachos</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Bastian Fromm</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Yin Lu</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Marc K. Halushka</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Michael Hackenberg</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Gianvito Urgese</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Elisa Ficarra</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Shruthi Bandyadka</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Jason Sydes</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Peter Batzel</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">John H. Postlethwait</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Phillipe Loher</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Eric Londin</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Aristeidis G. Telonis</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Isidore Rigoutsos</span>
    </span>, 
    <span itemscope itemprop="author" itemtype="http://schema.org/Person">
      <span itemprop="name">Lorena Pantano</span>
    </span>
    
  </div>
  

  <span class="article-date">
    
    <meta content="2018-06-27 13:13:02 -0900 -0900" itemprop="datePublished">
    <time datetime="2018-06-27 13:13:02 -0900 -0900" itemprop="dateModified">
      Jun 27, 2018
    </time>
  </span>
  <span itemscope itemprop="publisher" itemtype="http://schema.org/Person">
    <meta itemprop="name" content="Lorena Pantano">
  </span>

  

  
  

  

  
  
<div class="share-box" aria-hidden="true">
  <ul class="share">
    <li>
      <a class="twitter"
         href="https://twitter.com/intent/tweet?text=miRTOP%3a%20An%20open%20source%20community%20project%20for%20the%20development%20of%20a%20unified%20format%20file%20for%20miRNA%20data&amp;url=http%3a%2f%2flpantano.github.io%2ftalk%2fbosc2018%2f"
         target="_blank" rel="noopener">
        <i class="fab fa-twitter"></i>
      </a>
    </li>
    <li>
      <a class="facebook"
         href="https://www.facebook.com/sharer.php?u=http%3a%2f%2flpantano.github.io%2ftalk%2fbosc2018%2f"
         target="_blank" rel="noopener">
        <i class="fab fa-facebook-f"></i>
      </a>
    </li>
    <li>
      <a class="linkedin"
         href="https://www.linkedin.com/shareArticle?mini=true&amp;url=http%3a%2f%2flpantano.github.io%2ftalk%2fbosc2018%2f&amp;title=miRTOP%3a%20An%20open%20source%20community%20project%20for%20the%20development%20of%20a%20unified%20format%20file%20for%20miRNA%20data"
         target="_blank" rel="noopener">
        <i class="fab fa-linkedin-in"></i>
      </a>
    </li>
    <li>
      <a class="weibo"
         href="http://service.weibo.com/share/share.php?url=http%3a%2f%2flpantano.github.io%2ftalk%2fbosc2018%2f&amp;title=miRTOP%3a%20An%20open%20source%20community%20project%20for%20the%20development%20of%20a%20unified%20format%20file%20for%20miRNA%20data"
         target="_blank" rel="noopener">
        <i class="fab fa-weibo"></i>
      </a>
    </li>
    <li>
      <a class="email"
         href="mailto:?subject=miRTOP%3a%20An%20open%20source%20community%20project%20for%20the%20development%20of%20a%20unified%20format%20file%20for%20miRNA%20data&amp;body=http%3a%2f%2flpantano.github.io%2ftalk%2fbosc2018%2f">
        <i class="fas fa-envelope"></i>
      </a>
    </li>
  </ul>
</div>


  

</div>

    







  








<div class="btn-links mb-3">
  
  







  
    
  


<a class="btn btn-outline-primary my-1 mr-1" href="https://f1000research.com/slides/7-953#" target="_blank" rel="noopener">
  PDF
</a>











  
  
    
  
<a class="btn btn-outline-primary my-1 mr-1" href="https://f1000research.com/slides/7-953" target="_blank" rel="noopener">
  Slides
</a>







</div>


  
</div>



  <div class="article-container">

    
      <h3>Abstract</h3>
      <p class="pub-abstract" itemprop="text">MicroRNAs (miRNAs) are small RNA molecules (20-27 nt long) that are involved in eukaryotic  gene regulation. They regulate targeted genes by RNA complementarity, generally between the miRNA seed region and the 3’ UTR of messenger RNAs. They have become relevant to the study of developmental stages and diseases. With the advance of sequencing technology, the cost of detecting the miRNA transcriptome has decreased substantially, and has led to the discovery of isomiRs (with slight sequence variations relative to the annotated miRNAs). As a consequence, the amount of miRNA data has increased exponentially in the last 5 years, together with the number of pipelines to analyze it. However, there is still a lack of consensus standards in storing and sharing such data, generating diverse output files; hindering comparison between tools, data sharing, and development of downstream analyses independent of the tools used for miRNA detection and quantification. Here, we present a community based, open source project to work toward the standardization of miRNA pipelines and encourage the development of downstream tools for visualization, differential expression, sample clustering, and model prediction analyses. This project is an international collaboration, with experts from different countries that have been developing miRNA pipelines and resources. We have described a standard format for the output of miRNA detection and quantification tools using small RNA-seq data. The format is based on the GFF3 standard in order to support reference coordinates and parent/child relationships between the features. The format contains information foreach sequence and its annotation to the miRNA precursor, the definition of reference or isomiR sequence, its quality, the isomiR type, and abundance in the data set. Moreover, we support a command line python tool to manage the miRNA GFF3 format (miRTop). Currently, miRTop can convert the output of commonly used small RNA-Seq pipelines, such as seqbuster, isomiR-SEA, sRNAbench, and Prost!, as well as BAM files to the miRNA GFF3 format. Importantly, the miRge pipeline has adapted the GFF3 format natively. miRTop can convert miRNA GFF3 files to a count matrix that can be easily imported to any downstream tool (i.e. for differential expression analysis). Furthermore, the tool can export the miRNA GFF file to the isomiRs package, improving the usability of this tool and rendering it independent of the upstream methods used for quantifying miRNA sequences. Together, we believe that the miRTop project will improve accessibility and uniformity of the results oft miRNA pipelines, the possibility to easily run benchmarking analyses, and promote the development of miRNA tools downstream of the detection and quantification steps.</p>
    

    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Date</div>
          <div class="col-12 col-md-9" itemprop="datePublished">
            
            Jun 27, 2018
            <div class="talk-time">
              
                1:41 PM
                
                  &mdash; 2:00 PM
                
              
            </div>
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>

    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Event</div>
          <div class="col-12 col-md-9">
            <a href="https://gccbosc2018.sched.com/" target="_blank" rel="noopener">
            Bioinformatics Open Source Conference 2018
            </a>
          </div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>

    
    <div class="row">
      <div class="col-md-1"></div>
      <div class="col-md-10">
        <div class="row">
          <div class="col-12 col-md-3 pub-row-heading">Location</div>
          <div class="col-12 col-md-9">Portland, OR</div>
        </div>
      </div>
      <div class="col-md-1"></div>
    </div>
    <div class="d-md-none space-below"></div>
    

    <div class="space-below"></div>

    <div class="article-style">
      
    </div>

    


<div class="article-tags">
  
  <a class="badge badge-light" href="/tags/mirna/">miRNA</a>
  
  <a class="badge badge-light" href="/tags/isomir/">isomiR</a>
  
</div>



    



  







  </div>
</div>



<div class="container">
  <footer class="site-footer">
  

  <p class="powered-by">
    &copy; 2018 &middot; 

    Powered by the
    <a href="https://sourcethemes.com/academic/" target="_blank" rel="noopener">Academic theme</a> for
    <a href="https://gohugo.io" target="_blank" rel="noopener">Hugo</a>.

    
    <span class="float-right" aria-hidden="true">
      <a href="#" id="back_to_top">
        <span class="button_icon">
          <i class="fas fa-chevron-up fa-2x"></i>
        </span>
      </a>
    </span>
    
  </p>
</footer>

</div>


<div id="modal" class="modal fade" role="dialog">
  <div class="modal-dialog">
    <div class="modal-content">
      <div class="modal-header">
        <h5 class="modal-title">Cite</h5>
        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
          <span aria-hidden="true">&times;</span>
        </button>
      </div>
      <div class="modal-body">
        <pre><code class="tex hljs"></code></pre>
      </div>
      <div class="modal-footer">
        <a class="btn btn-outline-primary my-1 js-copy-cite" href="#" target="_blank">
          <i class="fas fa-copy"></i> Copy
        </a>
        <a class="btn btn-outline-primary my-1 js-download-cite" href="#" target="_blank">
          <i class="fas fa-download"></i> Download
        </a>
        <div id="modal-error"></div>
      </div>
    </div>
  </div>
</div>

    

    
    

    
    
    
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js" integrity="sha512-+NqPlbbtM1QqiK8ZAo4Yrj2c4lNQoGv8P79DPtKzj++l5jnN39rHA/xsqn8zE9l0uSoxaCdrOgFs6yjyfbBxSg==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.imagesloaded/4.1.3/imagesloaded.pkgd.min.js" integrity="sha512-umsR78NN0D23AzgoZ11K7raBD+R6hqKojyBZs1w8WvYlsI+QuKRGBx3LFCwhatzBunCjDuJpDHwxD13sLMbpRA==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha256-VsEqElsCHSGmnmHXGQzvoWjWwoznFSZc6hs7ARLRacQ=" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery.isotope/3.0.4/isotope.pkgd.min.js" integrity="sha512-VDBOIlDbuC4VWxGJNmuFRQ0Li0SKkDpmGyuhAG5LTDLd/dJ/S0WMVxriR2Y+CyPL5gzjpN4f/6iqWVBJlht0tQ==" crossorigin="anonymous"></script>
      <script src="https://cdnjs.cloudflare.com/ajax/libs/fancybox/3.2.5/jquery.fancybox.min.js" integrity="sha256-X5PoE3KU5l+JcX+w09p/wHl9AzK333C4hJ2I9S5mD4M=" crossorigin="anonymous"></script>

      
        
        <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/highlight.min.js" integrity="sha256-/BfiIkHlHoVihZdc6TFuj7MmJ0TWcWsMXkeDFwhi0zw=" crossorigin="anonymous"></script>
        
      

      
      
    

    
    

    
    
    

    
    
    <script>hljs.initHighlightingOnLoad();</script>
    

    
    
    <script>
      const search_index_filename = "/index.json";
      const i18n = {
        'placeholder': "Search...",
        'results': "results found",
        'no_results': "No results found"
      };
      const content_type = {
        'post': "Posts",
        'project': "Projects",
        'publication' : "Publications",
        'talk' : "Talks"
        };
    </script>
    

    
    

    
    
    <script id="search-hit-fuse-template" type="text/x-template">
      <div class="search-hit" id="summary-{{key}}">
      <div class="search-hit-content">
        <div class="search-hit-name">
          <a href="{{relpermalink}}">{{title}}</a>
          <div class="article-metadata search-hit-type">{{type}}</div>
          <p class="search-hit-description">{{snippet}}</p>
        </div>
      </div>
      </div>
    </script>
    

    
    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/fuse.js/3.2.1/fuse.min.js" integrity="sha256-VzgmKYmhsGNNN4Ph1kMW+BjoYJM2jV5i4IlFoeZA9XI=" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/mark.js/8.11.1/jquery.mark.min.js" integrity="sha256-4HLtjeVgH0eIB3aZ9mLYF6E8oU5chNdjU6p6rrXpl9U=" crossorigin="anonymous"></script>
    

    
    

    
    
    
    
    
    
    
    
    <script src="/js/academic.min.d037ee5294b166a79dec317c58aea9cc.js"></script>

    

  </body>
</html>

